src/languages/en.js

```diff
@@ -224,6 +224,10 @@ export default {
         setMyTimezoneAutomatically: 'Set my timezone automatically',
         timezone: 'Timezone',
         growlMessageOnSave: 'Your profile was successfully saved',
+        error: {
+            messageOn400: 'First name shouldn\'t be longer than 50 characters',
+            messageOn401: 'Last name shouldn\'t be longer than 50 characters',
+        },
```

Comment:
Before we consider merging, I've found that we also update first & last name via the `RequestCallPage` form. The back-end tries to update first and last name personal details. So I'm thinking we should instead store these errors in a new property, `personalDetails` like so:

```
profilePage: { ... },
personalDetails: {
    error: {
        firstNameLength: 'First name shouldn\'t be longer than 50 characters',
        lastNameLength: 'Last name shouldn\'t be longer than 50 characters',
    },
},
```